bodhi/server/models.py

```diff
@@ -4164,17 +4164,19 @@ def close_bug(self, update):
         ])
         bugs.bugtracker.close(self.bug_id, versions=versions, comment=self.default_message(update))
 
-    def modified(self, update):
+    def modified(self, update, comment):
         """
         Change the status of this bug to MODIFIED unless it is a parent security bug.
 
+        Also, comment on the bug stating that an update has been submitted.
+
         Args:
             update (Update): The update that is associated with this bug.
         """
         if update.type is UpdateType.security and self.parent:
             log.debug('Not modifying on parent security bug %s', self.bug_id)
         else:
-            bugs.bugtracker.modified(self.bug_id)
+            bugs.bugtracker.modified(self.bug_id, comment)
```

Comment:
I think it would be nice to follow what is done in the `testing` method and set the comment value here. 
That way the `modified` method in models.py does not require an extra `comment` argument

See https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L4156

Reply:
I see the use of a config template ```config['testing_bug_msg']``` in the ```default_message``` method called by ```testing```. Is that reusable? Or we need to define a new template for ```modified```?

Reply:
hum yeah good point I thought that the bug.default_message was using the `config['testing_bug_msg']` template. So let's keep this the way you have done it :+1: